.vscode/settings.json

```diff
@@ -0,0 +1,53 @@
+{
+	/* formatting */
+	"editor.defaultFormatter": "biomejs.biome",
+	"[javascript]": {
+		"editor.defaultFormatter": "esbenp.prettier-vscode"
+	},
+	"[typescript]": {
+		"editor.defaultFormatter": "esbenp.prettier-vscode"
+	},
+	"[javascriptreact]": {
+		"editor.defaultFormatter": "esbenp.prettier-vscode"
+	},
+	"[typescriptreact]": {
+		"editor.defaultFormatter": "esbenp.prettier-vscode"
+	},
+	"[xml]": {
+		"editor.defaultFormatter": "redhat.vscode-xml"
+	},
+	"[snippets]": {
+		"editor.defaultFormatter": "vscode.json-language-features"
+	},
+
+	/* recommended extension config */
+	"template-string-converter.addBracketsToProps": true,
+	"template-string-converter.autoRemoveTemplateString": true,
+	"stylelint.validate": ["css", "scss", "typescript", "typescriptreact"],
+	"cSpell.enabled": true,
+	"cSpell.diagnosticLevel": "Hint",
+
+	/* workbench settings */
+	"typescript.tsdk": "./node_modules/typescript/lib",
+	"typescript.enablePromptUseWorkspaceTsdk": true,
+	"typescript.preferences.autoImportFileExcludePatterns": [
+		"restyle",
+		"styled-components",
+		"styled-jsx"
+	],
+	"typescript.preferences.autoImportSpecifierExcludeRegexes": [
+		"^next/router$",
+		"^gsap/(?!all$).+",
+		"^gsap$"
+	],
+	"workbench.editor.customLabels.patterns": {
+		"**/index.tsx": "${dirname} / index.tsx",
+		"**/index.ts": "${dirname} / index.ts",
+		"**/page.tsx": "${dirname} / page.tsx",
+		"**/layout.ts": "${dirname} / layout.ts"
```

Comment:
this should be `layout.tsx` actually

we probably need to fix this in the starter too lol